wallet/walletrpc.c

```diff
@@ -1005,3 +1007,61 @@ static const struct json_command sendpsbt_command = {
 };
 
 AUTODATA(json_command, &sendpsbt_command);
+
+/* This is a dangerus command, but looks like that the dev rescan output is not covering
+ * a use case, where we have the output is marked as confirment but we want to move it to
+ * a spend confition. This usually do not happens but with dev rescan it can (still unclear how).
+ *
+ * So we need to interact with an external plugin in order to inject the corrent spend height
+ * inside the output.
+ *
+ * It can work is I rescan the blockchain? */
```

Comment:
Let's not expose this. It is here to address a transient error in our data model, that can trivially be resolved by `dev-rescan-outputs` as soon as it is fixed. Papering over is bad, as it teaches users to "just run this" and not report a real issue anymore.

Reply:
Well this is the only way that I found to fix the mess with my node, how I can fix the db problem with my node from `dev-rescan-outputs`.

>Papering over is bad, as it teaches users to "just run this" and not report a real issue anymore.

We are already saying this with `dev-rescan-outputs` are we?